cmd/table.go

```diff
@@ -43,15 +44,30 @@ func init() {
 		},
 
 		Run: func(_ *cobra.Command, args []string) {
-			if err := createTable(args, params); err != nil {
-				log.SetOutput(os.Stderr)
-				log.Println(err)
-				os.Exit(1)
+			log.SetOutput(os.Stderr)
```

Comment:
[nit] I believe that's the default ([see here](https://cs.opensource.google/go/go/+/refs/tags/go1.21.1:src/log/log.go;l=87))

Reply:
Ah, good! I'll have it removed 👍 

(I guess there's a theoretical chance some other command would have set this, but not in this case)